common/beerocks/bwl/shared/nl80211_client_impl.cpp

```diff
@@ -182,4 +182,32 @@ bool nl80211_client_impl::get_sta_info(const std::string &local_interface_name,
         });
 }
 
+bool nl80211_client_impl::set_tx_power_limit(const std::string &local_interface_name,
+                                             uint32_t limit)
+{
+    if (!m_socket) {
+        LOG(ERROR) << "Socket is NULL!";
+        return false;
+    }
+
+    // Get the interface index for given interface name
+    int iface_index = if_nametoindex(local_interface_name.c_str());
+    if (0 == iface_index) {
+        LOG(ERROR) << "Failed to read the index of interface " << local_interface_name << ": "
```

Comment:
I think that you should either validate that `local_interface_name` is not empty, or at least add `''` to the log (so that it'll be easier to spot an empty value):
```suggestion
        LOG(ERROR) << "Failed to read the index of interface '" << local_interface_name << "': "
```